cli/pcluster/config/validators.py

```diff
@@ -285,6 +285,14 @@ def dcv_enabled_validator(param_key, param_value, pcluster_config):
         if get_partition() not in get_supported_dcv_partition():
             errors.append("NICE DCV is not supported in the selected region '{0}'".format(get_region()))
 
+        master_instance_type = cluster_section.get_param_value("master_instance_type")
+        if re.search(r"(micro)|(nano)", master_instance_type):
```

Comment:
Do we want to add tests for this validator, including a couple of simple test cases to test the default and so on?

Reply:
Yes I'll do it.

Reply:
Done